guiEditor/src/diagram/workbench.tsx

```diff
@@ -66,7 +66,9 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     private _clipboard: Control[] = [];
     private _selectAll: boolean = false;
     private _camera: ArcRotateCamera;
-    private _cameraRadias = 1500;
+    private _cameraRadias: number;
+    private _cameraMaxRadiasFactor = 8192; // 2^13

```

Comment:
why did you select this number? shouldn't this be relative to the canvas size?

Reply:
Camera limitations.  I am thinking of lowering canvas size now to match.

Reply:
is there a max radius to the camera?

Reply:
apparently. Things glitch out after that. Just like adts stop working after 2^14

Reply:
apparently. Things glitch out after that. Just like adts stop working after 2^14

Reply:
can you show me a playground where that happens?

Reply:
did some updates :)